audio/Audio.py

```diff
@@ -0,0 +1,30 @@
+import pyaudio
+import wave
+
+
+class Audio:
+
+    def __init__(self, file):
+        self.file = file
+        self.CHUNK = 1024
```

Comment:
You may declare static constants directly in the class like
```
class Audio:

    CHUNK = 1024

    def __init__(self, file):
         self.file = file

    def play_audio...
```

And then reference it later in the code in the same way as you do now (using _self.CHUNK_). I think it might be cleaner with keeping the constants outside the constructor.